SQL.test.js

```diff
@@ -498,6 +498,28 @@ test('should be able to use nested SQLStatements in template literal', t => {
   t.end()
 })
 
+test('should be able to use the result of SQL.glue([SQL``, SQL``], separator) result with multiple values inside the first element of SQL.glue', t => {
+  const ids = [1, 2, 3]
+  const name = 'foo'
+  const inIds = SQL.glue(
+    ids.map((id) => SQL`${id}`),
+    ' , '
+  )
+  const condition = SQL`tsd.id IN (${inIds})`
+
+  const filters = [
+    condition,
+    SQL`tsd.name = ${name}`
+  ]
+
+  const sql = SQL`SELECT tsd.* FROM data tsd WHERE ${SQL.glue(filters, ' AND ')}`
+  t.equal(
+    sql.text,
```

Comment:
Are the `sql.replacement` in the correct order too?

Reply:
Check again, I added the sql.values and sql.debug assertions(to check replacement)